drivers/ovsdriver.go

```diff
@@ -377,44 +373,64 @@ func (d *OvsDriver) DeleteEndpoint(id string) (err error) {
 	return nil
 }
 
-// CreatePeerHost creates peer host object and adds VTEPs if necessary
-func (d *OvsDriver) CreatePeerHost(id string) error {
-	log.Infof("CreatePeerHost for %s", id)
+// AddPeerHost adds VTEPs if necessary
+func (d *OvsDriver) AddPeerHost(node core.ServiceInfo) error {
+	// Nothing to do if this is our own IP
+	if node.HostAddr == d.localIP {
+		return nil
+	}
+
+	log.Infof("CreatePeerHost for %+v", node)
```

Comment:
Ah, I forgot about that! Let’s fix these please since it’s trivial 
to get at them with this flag.

On 22 Sep 2015, at 13:46, Madhav Puri wrote:

>> @@ -377,44 +373,64 @@ func (d *OvsDriver) DeleteEndpoint(id string) 
>> (err error) {
>> 	return nil
>> }
>>
>> -// CreatePeerHost creates peer host object and adds VTEPs if 
>> necessary
>> -func (d *OvsDriver) CreatePeerHost(id string) error {
>> -	log.Infof("CreatePeerHost for %s", id)
>> +// AddPeerHost adds VTEPs if necessary
>> +func (d *OvsDriver) AddPeerHost(node core.ServiceInfo) error {
>> +	// Nothing to do if this is our own IP
>> +	if node.HostAddr == d.localIP {
>> +		return nil
>> +	}
>> +
>> +	log.Infof("CreatePeerHost for %+v", node)
>
> just for future reference, `netmaster` and `netplugin` take `--debug` 
> flag which if turned on will print all the debug messages as well.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/netplugin/pull/148/files#r40141694